### ISSUE TITLE
[Download Prices] crashes

### ISSUE THREAD

**Issue by tash** [EVEIPH/EVE-IPH#96]
Version: `4.0.6725.42163`
```
An Unhandled Exception has occured and EVE Isk per Hour will now close.

Please fill out the following information so I can reproduce the bug

What is your Operating System? 
What tab or screen did the error occur? 
What are the steps to reproduce the Error? 
Web link to a screenshot of your error: 
In addition to a screenshot, copy the data below and send to developer.

Source: Microsoft.VisualBasic
Message: Conversion from string "" to type 'Integer' is not valid.
Raw Error Text: System.InvalidCastException: Conversion from string "" to type 'Integer' is not valid. ---> System.FormatException: Die Eingabezeichenfolge hat das falsche Format.
   bei Microsoft.VisualBasic.CompilerServices.Conversions.ParseDouble(String Value, NumberFormatInfo NumberFormat)
   bei Microsoft.VisualBasic.CompilerServices.Conversions.ToInteger(String Value)
   --- Ende der internen Ausnahmestapelüberwachung ---
   bei Microsoft.VisualBasic.CompilerServices.Conversions.ToInteger(String Value)
   bei EVE_Isk_per_Hour.frmMain.UpdatePricesCache(List`1 CacheItems)
   bei EVE_Isk_per_Hour.frmMain.LoadPrices(List`1 SentItems)
   bei EVE_Isk_per_Hour.frmMain.btnImportPrices_Click(Object sender, EventArgs e)
   bei System.Windows.Forms.Control.OnClick(EventArgs e)
   bei System.Windows.Forms.Button.OnClick(EventArgs e)
   bei System.Windows.Forms.Button.OnMouseUp(MouseEventArgs mevent)
   bei System.Windows.Forms.Control.WmMouseUp(Message& m, MouseButtons button, Int32 clicks)
   bei System.Windows.Forms.Control.WndProc(Message& m)
   bei System.Windows.Forms.ButtonBase.WndProc(Message& m)
   bei System.Windows.Forms.Button.WndProc(Message& m)
   bei System.Windows.Forms.Control.ControlNativeWindow.OnMessage(Message& m)
   bei System.Windows.Forms.Control.ControlNativeWindow.WndProc(Message& m)
   bei System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)
Error Tracking:
```

**Comment by tash** [comment]
> What is your Operating System?

Microsoft Win 10 Home, 10.0.17134 Build 17134

> What tab or screen did the error occur?

[Update Prices]

> What are the steps to reproduce the Error?

[Select All Items] > [Download Prices]

> Web link to a screenshot of your error:
> In addition to a screenshot, copy the data below and send to developer.

see exception above

**Comment by Zifrian** [comment]
What system/region are you selecting? I'm not getting an error with the ones I'm testing it with.

**Comment by tash** [comment]
![Download Price Settings](https://i.imgur.com/fqhoBFG.png)

**Comment by tash** [comment]
[Single Select] seems to be working alright

**Comment by tash** [comment]
![grafik](https://user-images.githubusercontent.com/477569/40839301-3a4fe9f4-65a2-11e8-97a5-dbdbae47ac54.png)

gives me:
```
An Unhandled Exception has occured and EVE Isk per Hour will now close.

Please fill out the following information so I can reproduce the bug

What is your Operating System? 
What tab or screen did the error occur? 
What are the steps to reproduce the Error? 
Web link to a screenshot of your error: 
In addition to a screenshot, copy the data below and send to developer.

Source: System.Data.SQLite
Message: SQLite error
near " ": syntax error
Raw Error Text: System.Data.SQLite.SQLiteException (0x80004005): SQLite error
near " ": syntax error
   bei System.Data.SQLite.SQLite3.Prepare(SQLiteConnection cnn, String strSql, SQLiteStatement previous, UInt32 timeoutMS, String& strRemain)
   bei System.Data.SQLite.SQLiteCommand.BuildNextCommand()
   bei System.Data.SQLite.SQLiteCommand.GetStatement(Int32 index)
   bei System.Data.SQLite.SQLiteDataReader.NextResult()
   bei System.Data.SQLite.SQLiteDataReader..ctor(SQLiteCommand cmd, CommandBehavior behave)
   bei System.Data.SQLite.SQLiteCommand.ExecuteReader(CommandBehavior behavior)
   bei System.Data.SQLite.SQLiteCommand.ExecuteReader()
   bei EVE_Isk_per_Hour.MarketPriceInterface.UpdatableMarketData(TypeIDRegion Item, String CacheType)
   bei EVE_Isk_per_Hour.MarketPriceInterface.UpdateMarketOrders(List`1 CacheItems)
   bei EVE_Isk_per_Hour.frmMain.LoadPrices(List`1 SentItems)
   bei EVE_Isk_per_Hour.frmMain.btnImportPrices_Click(Object sender, EventArgs e)
   bei System.Windows.Forms.Control.OnClick(EventArgs e)
   bei System.Windows.Forms.Button.OnClick(EventArgs e)
   bei System.Windows.Forms.Button.OnMouseUp(MouseEventArgs mevent)
   bei System.Windows.Forms.Control.WmMouseUp(Message& m, MouseButtons button, Int32 clicks)
   bei System.Windows.Forms.Control.WndProc(Message& m)
   bei System.Windows.Forms.ButtonBase.WndProc(Message& m)
   bei System.Windows.Forms.Button.WndProc(Message& m)
   bei System.Windows.Forms.Control.ControlNativeWindow.OnMessage(Message& m)
   bei System.Windows.Forms.Control.ControlNativeWindow.WndProc(Message& m)
   bei System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)
Error Tracking: 
```

**Comment by Zifrian** [comment]
Thanks. I fixed this issue with Build 4.0.6726.17366
